2-mandatory/1-create-functions.js

```diff
@@ -33,7 +41,14 @@ Write a function that:
 - Returns a new array containing the same elements, but without the element at the passed index.
 */
 
-function remove() {
+function remove(arr, ind) {
+  let newarr = []
+  for(let i=0; i<arr.length; i++){
+    if(i!=ind){
+      newarr.push(arr[i])
+    }
+  }
+  return newarr
```

Comment:
there is a function that can do this more directly.
```
splice(start)
splice(start, deleteCount)
splice(start, deleteCount, item1)
splice(start, deleteCount, item1, item2, itemN)
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice

Reply:
yes, changed it to newArr.splice(ind,1)